swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+   def __init__(self, id = None, condition = 0, age = None):
+      self.id = uuid.uuid1().int if id is None else id
+      self.category = "Item"
+      self.condition = condition
+      self.age = age
+   
+   def get_category(self):
+      return self.category
```

Comment:
Instead of hardcoding an attribute `category` to the string value "Item" on line 5, we can write Python code here that will get an instance's class name. 

```suggestion
      return self.__class__.__name__
```

If you refactor `get_category`, then you can delete line 5 above and also the 3 places you overrode `get_category` in the child classes.